### PR TITLE
Fix/ext 166 online event display

### PR DESCRIPTION
**Pull request by paulmskim**
Ticket: https://moderntribe.atlassian.net/browse/EXT-166

### CODE REVIEW

File: src/Tribe/Template_Modifications.php [hunk 1]
```diff
@@ -130,12 +149,53 @@ public function add_archive_control_markers( $file, $name, $template ) {
...
+	 *
+	 * @since TBD
+	 *
+	 * @param string   $template_name Template name to insert into the html.
+	 * @param string   $html          HTML to be modified.
+	 * @param string   $file          Complete path to include the PHP File.
+	 * @param array    $name          Template name.
+	 * @param Template $template      Current instance of the Template.
+	 *
+	 * @return void
```
Comment (jesseeproductions) [inline src/Tribe/Template_Modifications.php]: Please change to return string with a description.

**Review by jesseeproductions — COMMENTED** [review]
Looks good, I rearranged the css properties and removed a file I included, but was not longer used.

File: src/Tribe/Template_Modifications.php (hunk 1 above)
Comment (jesseeproductions) [reply src/Tribe/Template_Modifications.php]: And please add a readme entry, then all set.

**Comment by paulmskim** [comment]
waiting on @bordoni to make changes for the regex before merge.

**Review by bordoni — APPROVED** [review]

**Pull request merged**